src/frontend/src/common/components/library/data_subview/components/EditSamplesConfirmationModal/index.tsx

```diff
@@ -32,6 +41,35 @@ const EditSamplesConfirmationModal = ({
   onClose,
   open,
 }: Props): JSX.Element | null => {
+  const [isValid, setIsValid] = useState(false);
+  const [metadata, setMetadata] =
+    useState<SampleIdToEditMetadataWebform | null>(null);
+
+  const { data: namedLocationsData } = useNamedLocations();
+  const namedLocations =
+    namedLocationsData?.namedLocations || ([] as NamedGisaidLocation[]);
+
+  function structureInitialMetadata(item: Sample): SampleEditMetadataWebform {
+    const i = _.pick(
+      item,
+      Object.keys(SAMPLE_EDIT_WEBFORM_METADATA_KEYS_TO_HEADERS)
+    ) as SampleEditMetadataWebform;
+    const cl = i.collectionLocation;
+    if (cl) {
+      cl.name = `${cl.region}/${cl.country}/${cl.division}/${cl.location}`;
```

Comment:
I'm not sure if there is anything to be done here, but this code was slightly confusing to me on first read through. ("what's happening with cl? it isn't be returned/used")

Reply:
what would make it less confusing?

Reply:
I guess it's fine the way it is. I tried a few other more explicit options locally, but none seemed that satisfying